src/macros.rs

```diff
@@ -1,3 +1,4 @@
+/// Log trace message (feature: enabled).
```

Comment:
@gendx as a sidenote, these macros end up in the public API, but I'm not sure whether you really intended that. If you are planning a breaking release soon, you may consider taking them out.

Reply:
Indeed. The public API should just consist of the `enable_logging` feature, these macros should only be useful internally (and indeed I don't want any dependency to rely on them, which would add maintenance burden).